src/ensembl/src/content/app/entity-viewer/state/bookmarks/entityViewerBookmarksSlice.ts

```diff
@@ -14,13 +14,26 @@
  * limitations under the License.
  */
 
-import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit';
+import merge from 'lodash/merge';
+
+import { getPreviouslyViewedEntities } from './entityViewerBookmarksSelectors';
+
 import entityViewerBookmarksStorageService from 'src/content/app/entity-viewer/services/bookmarks/entity-viewer-bookmarks-storage-service';
+import entityViewerStorageService from 'src/content/app/entity-viewer/services/entity-viewer-storage-service';
+
+import {
+  Filters,
+  SortingRule
+} from 'src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice';
+import { RootState } from 'src/store';
 
 type PreviouslyViewedEntity = {
   entity_id: string;
   label: string | string[];
   type: 'gene';
+  filters?: Filters;
+  sortingRule?: SortingRule;
```

Comment:
These two fields will no longer be necessary according to this PR's logic, I believe?

Reply:
Yes, I've removed them 👍